### PR TITLE
[vcpkg baseline][lensfun] Fix and cleanup

### PR DESCRIPTION
**Pull request by dg0yt**
Fix x64-linux baseline regression dependent on installation order.
Enable x64 optimizations.
Update license information.
Cleanup.
Remove the python module from the default features.

### CODE REVIEW

File: ports/lensfun/vcpkg.json [hunk 1]
```diff
@@ -16,5 +16,21 @@
...
       "host": true
     }
-  ]
+  ],
+  "default-features": [
+    {
+      "name": "python",
+      "platform": "native"
+    }
+  ],
```
Comment (dg0yt) [inline ports/lensfun/vcpkg.json:25]: I would be glad to remove it from default features.

**Comment by dg0yt** [comment]
Linux:
~~~
tools/python3/python3
tools/python3/python3.11
~~~
Windows:
~~~
tools/python3/python.exe
~~~
:angry:

File: ports/lensfun/vcpkg.json [hunk 2]
```diff
@@ -16,5 +16,15 @@
       "name": "vcpkg-cmake-config",
       "host": true
     }
-  ]
+  ],
+  "features": {
+    "python": {
+      "description": "Build python module",
+      "supports": "native & !windows",
```
Comment (dg0yt) [inline ports/lensfun/vcpkg.json:23]: Windows didn't build the python module, according to the previous CI file list, so this is not a regression. And now not possible due to https://github.com/microsoft/vcpkg/pull/36354#issuecomment-1909482362.

File: ports/lensfun/vcpkg.json (hunk 1 above)
Comment (dg0yt) [reply ports/lensfun/vcpkg.json]: Done.

**Comment by Neumann-A** [comment]
> 😠

Let me tell you there are more problems. I added the python3 binary so that scripts executed with `/usr/bin/env python3` can actually find the `python3` build by vcpkg.

**Comment by dg0yt** [comment]
> I added the python3 binary so that scripts executed with `/usr/bin/env python3` can actually find the `python3` build by vcpkg.

Which is probably the cause of the installation order defects like `lensfun` here. As long as vcpkg.cmake sets the program path regardless of dependencies.

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thank you!

**Review by JavierMatosD — APPROVED** [review]

**Pull request merged**